templates/primary-secondary/primary-secondary.js

```diff
@@ -27,6 +28,11 @@ function isMobile() {
 	return matchMedia('only screen and (max-width: 767px)').matches;
 }
 
+function isWindows() {
+	const val = window.navigator.userAgent.indexOf('Windows') > -1;
```

Comment:
Nit: this could just be:
```javascript
const isWindows = window.navigator.userAgent.indexOf('Windows') > -1;
```
... so that it doesn't need to be evaluated for every `render` call.

Reply:
Good call -- updated.